extensions/amp-selector/1.0/selector.js

```diff
@@ -29,15 +29,17 @@ const SelectorContext = Preact.createContext(
 export function Selector({
   as: Comp = 'div',
   disabled,
+  defaultValue = [],
   value,
   multiple,
   onChange,
   role = 'listbox',
   children,
   ...rest
 }) {
-  const [selectedState, setSelectedState] = useState(value ? value : []);
-  // TBD: controlled values require override of properties.
+  const [selectedState, setSelectedState] = useState(
+    value ? value : defaultValue
```

Comment:
Is `value` always an array? Could it be a number, e.g. `0`? If so, it might be better to switch to `??`

Reply:
`value` is always an array. I agree it would be nicer for single selectors to accept a non-array `value`, but that should probably be in a separate change as there are more implications for the internals.